kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
We always add `from __future__ import absolute_import` to every file, the build will break if not, so it shouldn't be a problem